src/components/Garden/Stake/SideBar.tsx

```diff
@@ -27,7 +37,7 @@ function Sidebar({ stakeActions, staking, token, onDepositOrWithdraw }) {
         stakeActions={stakeActions}
         total={total}
         tokenAddress={token.id}
-        tokenDecimals={token.decimals}
+        tokenDecimals={token?.decimals}
```

Comment:
```suggestion
        tokenDecimals={token.decimals}
```
I think its not necessary, right?

Reply:
I think it's not. Will change it. Thanks.

Reply:
Fixed!